### PR TITLE
Add Inter as new font option

### PR DESCRIPTION
**Pull request by felixfischer**
Inter is a modern high-quality font that makes a solid impression in a CV. It is carefully crafted & designed for computer screens. It features a tall x-height to aid in readability of mixed-case and lower-case text.

### CODE REVIEW

**Comment by xitanggg** [comment]
Thanks @felixfischer for creating this PR. My thinking is to limit the options to 5 Sans Serif fonts, so all Sans Serif fonts remain on the same line. Adding another one would cause the font selection to wrap and not so great UI/UX. 
image

Also, fyi, adding a new font is a bit tricky. I have updated and added more descriptions at `src/app/components/fonts/constants.ts` that outlines this problem
> One major problem with adding a new font family is that most font family doesn't work with React PDF out of box. The texts would appear fine in the PDF, but copying and pasting them would result in different texts.
> A solution to this problem is to import and re-export the font with a font editor, e.g. fontforge or birdfont.
Basically, you have to import and re-export the font with a font editor for it to work. Otherwise, it returns some gibberish (see description in image below)
image

For this reason, I wasn't able to say add an option for people to choose any google fonts or upload their own fonts to use. This would the best option to give the best flexibility. For now, I think it makes sense to keep the options to a selected set as it is easier to maintain

**Comment by felixfischer** [comment]
OK. Thanks for taking the time to answer, though! :)

**Comment by xitanggg** [comment]
Thanks for understanding! :)
